extensions/ql-vscode/src/remote-queries/variant-analysis-manager.ts

```diff
@@ -89,6 +108,11 @@ export class VariantAnalysisManager extends DisposableObject implements VariantA
     await this.variantAnalysisResultsManager.loadResults(variantAnalysisId, repositoryFullName);
   }
 
+  private async variantAnalysisRecordExists(variantAnalysisId: number): Promise<boolean> {
+    const filePath = path.join(this.storagePath, variantAnalysisId.toString());
```

Comment:
I think it would be useful to extract this to a separate method which can also be used for getting the query directory, like also mentioned by @charisk in https://github.com/github/vscode-codeql/pull/1622#discussion_r999280135.

Reply:
👍 agreed, but I'm personally happy for this PR to be merged without that (and look at tidying that up later)

Reply:
will be done here: https://github.com/github/vscode-codeql/pull/1622#discussion_r999327527

Reply:
This can now use `getVariantAnalysisStorageLocation`
```suggestion
    const filePath = this.getVariantAnalysisStorageLocation(variantAnalysisId);
```